src/block_coordinate_algorithms.jl

```diff
@@ -267,24 +286,29 @@ function perform_bc_updates(bc_algo::BCFW, f, grad!, lmo, x0)
                     1.0,
                     linesearch_workspace,
                     memory_mode,
-                )      
-                
+                )
+
                 x = muladd_memory_mode(memory_mode, x, gamma, d)
             end
         end
 
-        
+
         # go easy on the memory - only compute if really needed
         if (
             (mod(t, print_iter) == 0 && verbose) ||
             callback !== nothing ||
             line_search isa Shortstep
         )
-            infeas = sum([fast_dot(selectdim(x,ndim,i) - selectdim(x,ndim,j), selectdim(x,ndim,i) - selectdim(x,ndim,j)) for i in 1:l for j in 1:i-1])
+            infeas = sum([
+                fast_dot(
+                    selectdim(x, ndim, i) - selectdim(x, ndim, j),
+                    selectdim(x, ndim, i) - selectdim(x, ndim, j),
+                ) for i in 1:l for j in 1:i-1
+            ])
```

Comment:
```suggestion
            infeas = sum(
                fast_dot(
                    selectdim(x, ndim, i) - selectdim(x, ndim, j),
                    selectdim(x, ndim, i) - selectdim(x, ndim, j),
                ) for i in 1:l for j in 1:i-1
            )
```
non-allocating version